steps/version-extractor-android/0.0.1/step.yml

```diff
@@ -0,0 +1,56 @@
+title: Get versionName and versionCode on Android
+summary: |
+  Extract the versionName and the versionCode on an Android project using a Gradle task
+description: |
+  Extracts the Name version and the Code version from the Android project. It is useful if the versionName and versionCode in `build.gradle` or `build.gradle.kts` are defined somewhere else. Therefore, this step uses a Gradle task to print these two values.
```

Comment:
Name version -> version name and similarly for code.
It is important that this step reads only values from default config which may be different from actually built variant.

Reply:
I'm not sure If I'm getting it right. So, reading the defaultConfig values is right then, isn't it? For example, I'm using `android.`**`defaultConfig`**`.versionName` for the version name which always reads values from default config and not from a particular build variant.

Or maybe I should add an input field where user can define the build variant?

Reply:
I meant that if it is not specified in docs which variant is used then it may be unclear for users. Eg. if they are building only one of the variants which uses different values.

Additional input field for variant sounds good. Field may may be optional and default to defualt config if empty.